gradle/buildSrc/src/main/kotlin/io/spine/gradle/internal/deps.kt

```diff
@@ -124,55 +112,189 @@ object Versions {
+    const val testLib = "com.google.guava:guava-testlib:${version}"
+}
+
+// https://github.com/grpc/grpc-java
+object Grpc {
+    @Suppress("MemberVisibilityCanBePrivate")
+    const val version     = "1.35.0"
+    const val core        = "io.grpc:grpc-core:${version}"
+    const val stub        = "io.grpc:grpc-stub:${version}"
+    const val okHttp      = "io.grpc:grpc-okhttp:${version}"
+    const val protobuf    = "io.grpc:grpc-protobuf:${version}"
+    const val netty       = "io.grpc:grpc-netty:${version}"
+    const val nettyShaded = "io.grpc:grpc-netty-shaded:${version}"
+    const val context     = "io.grpc:grpc-context:${version}"
+}
+
+// https://junit.org/junit5/
+object JUnit {
+    private const val version            = "5.7.0"
+    private const val legacyVersion      = "4.13.1"
+    private const val apiGuardianVersion = "1.1.0"
+    private const val pioneerVersion     = "1.0.0"
+    private const val platformVersion    = "1.7.0"
+
+    const val legacy = "junit:junit:${legacyVersion}"
+    val api = listOf(
+        "org.junit.jupiter:junit-jupiter-api:${version}",
+        "org.junit.jupiter:junit-jupiter-params:${version}",
+        "org.apiguardian:apiguardian-api:${apiGuardianVersion}"
     )
-    val protoc                 = "com.google.protobuf:protoc:${Versions.protobuf}"
-    val googleHttpClient       = "com.google.http-client:google-http-client:${Versions.httpClient}"
-    val googleHttpClientApache = "com.google.http-client:google-http-client-apache:${Versions.apacheHttpClient}"
-    val appengineApi           = "com.google.appengine:appengine-api-1.0-sdk:${Versions.appengineApi}"
-    val firebaseAdmin          = "com.google.firebase:firebase-admin:${Versions.firebaseAdmin}"
-    val jacksonDatabind        = "com.fasterxml.jackson.core:jackson-databind:${Versions.jackson}"
-    val roasterApi             = "org.jboss.forge.roaster:roaster-api:${Versions.roaster}"
-    val roasterJdt             = "org.jboss.forge.roaster:roaster-jdt:${Versions.roaster}"
-    val animalSniffer          = "org.codehaus.mojo:animal-sniffer-annotations:${Versions.animalSniffer}"
-    val ci = "true".equals(System.getenv("CI"))
+    const val runner  = "org.junit.jupiter:junit-jupiter-engine:${version}"
+    const val pioneer = "org.junit-pioneer:junit-pioneer:${pioneerVersion}"
+    const val platformCommons = "org.junit.platform:junit-platform-commons:${platformVersion}"
+}
+
+// https://github.com/JetBrains/kotlin
+// https://github.com/Kotlin
+object Kotlin {
+    @Suppress("MemberVisibilityCanBePrivate") // used directly from outside
+    const val version      = "1.4.21"
+    const val reflect      = "org.jetbrains.kotlin:kotlin-reflect:${version}"
+    const val stdLib       = "org.jetbrains.kotlin:kotlin-stdlib:${version}"
+    const val stdLibCommon = "org.jetbrains.kotlin:kotlin-stdlib-common:${version}"
+    const val stdLibJdk8   = "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${version}"
```

Comment:
If we're going to use the `kotlin` Gradle plugin, we may declare these dependencies as follows:
```kotlin
dependencies {
    implementation(kotlin("stdlib"))
    implementation(kotlin("reflect"))
    implementation(kotlin("<your module here>"))
}
```

The version is still needed to use in the plugin declaration statement.